### PR TITLE
feat: Boring p24

### LINKED ISSUE #952: feat: Boring p24
Let's make this pattern:
image

Please work at this file: https://github.com/TroyTae/game-of-life/blob/master/src/life/oscillator/boring-p24.json

If you wanna tackle this issue, please leave comment.  
I'll assign to you!  

Are you first time for contributing this repo?  
Please read this [guide](https://github.com/TroyTae/game-of-life/blob/master/.github/CONTRIBUTING.md) :)

### PR DESCRIPTION
**Pull request by Axhat**
Added the required pattern in the desired file. Used the tool for filling
the pattern.

resolve #952 feat: Boring p24

### CODE REVIEW

File: src/life/oscillator/boring-p24.json [hunk 1]
```diff
@@ -1,8 +1,24 @@
 {
   "title": "Boring p24",
-  "life": [
-    [0, 0, 0],
-    [0, 1, 0],
-    [0, 0, 0]
-  ]
+  "life": [[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
```
Comment (TroyTae) [inline src/life/oscillator/boring-p24.json:3]: Please recover enter of this line

Comment (Axhat) [reply src/life/oscillator/boring-p24.json:3]: can you please elaborate the issue? i am finding this information insufficient.

Comment (TroyTae) [reply src/life/oscillator/boring-p24.json:3]: You need to fix two things:

1. Now you remove first line
image
This is the original code.
Just insert new lint before first row.

2. Almost pattern is good, but you miss some pattern.
Please compare your code and images (I indicate pattern you missed as red circle)

Comment (Axhat) [reply src/life/oscillator/boring-p24.json:3]: Thanks, yes i noticed that i am missing some of pattern.
Thanks for helping out.

**Comment by Axhat** [comment]
Done

**Pull request merged**